tests/api.c

```diff
@@ -33265,7 +33265,753 @@ static void test_wolfSSL_EVP_PKEY_assign(void)
     printf(resultFmt, passed);
 #endif /* OPENSSL_ALL */
 }
+static void test_wolfSSL_EVP_PKEY_base_id(void)
+{
+#if defined(OPENSSL_ALL)
+    WOLFSSL_EVP_PKEY* pkey;
+
+    printf(testingFmt, "wolfSSL_EVP_PKEY_base_id");
+
+    AssertNotNull(pkey = wolfSSL_EVP_PKEY_new());
+
+    AssertIntEQ(wolfSSL_EVP_PKEY_base_id(NULL), NID_undef);
+
+    AssertIntEQ(wolfSSL_EVP_PKEY_base_id(pkey), 6);
+
+    EVP_PKEY_free(pkey);
+
+    printf(resultFmt, passed);
+#endif
+}
+static void test_wolfSSL_EVP_PKEY_id(void)
+{
+#if defined(OPENSSL_ALL)
+    WOLFSSL_EVP_PKEY* pkey;
+
+    printf(testingFmt, "wolfSSL_EVP_PKEY_id");
+
+    AssertNotNull(pkey = wolfSSL_EVP_PKEY_new());
+
+    AssertIntEQ(wolfSSL_EVP_PKEY_id(NULL), 0);
+
+    AssertIntEQ(wolfSSL_EVP_PKEY_id(pkey), 6);
```

Comment:
Same as above, please replace "6" with a meaningful enum/define.  We try our best to avoid using "magic numbers" when possible.